src/elements-experimental/seat-reservation/seat-reservation-navigation-coach/seat-reservation-navigation-coach.scss

```
@@ -36,7 +40,7 @@ $disabled: '[disabled], :disabled';
     }
 
     .sbb-sr-navigation--first-class {
-      margin-inline-start: var(--sbb-seat-reservation-navigation-travelclass-spacing);
+      margin-inline-start: var(--sbb-reservation-navigation-travelclass-spacing);
```

please keep the whole prefix for css variables (they bleed through shadow DOM). This would be valid also for all other occurrences.
```suggestion
      margin-inline-start: var(--sbb-seat-reservation-navigation-travelclass-spacing);
```

---

We had a mixture of "--sbb-seat-reservation" and "--sbb-reservation" in the past. Unfortunately, the second variant is already in use in an application which use our component.. Therefore we ave standardized this to have all custom properties in the same way. We would like to avoid a breaking change with the new version if we would change every custom property

---

Ok, we will change it. Even this will mean that we are introducing a major breaking-change due to this. :(